### PR TITLE
Update OpenCvSharp4

### PR DESCRIPTION
**Pull request by sharwell**
Fixes NuGet warnings regarding package compatibility with the current target framework.

### CODE REVIEW

File: src/VAP/BGSObjectDetector/BGSObjectDetector.csproj [hunk 1]
```diff
@@ -1,11 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.2</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenCvSharp4" Version="4.0.0.20190108" />
+    <PackageReference Include="OpenCvSharp4" Version="4.1.1.20191110" />
```
Comment (sharwell) [inline src/VAP/BGSObjectDetector/BGSObjectDetector.csproj]: 📝 I'm planning to refactor all the version references to a **Versions.props** file so they are centrally-managed for this repository, but I did not make this change as part of this pull request.

File: src/VAP/BGSObjectDetector/BGSObjectDetector.csproj [hunk 2]
```diff
@@ -1,11 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (sharwell) [inline src/VAP/BGSObjectDetector/BGSObjectDetector.csproj:1]: 📝 Visual Studio always wants to change these files to UTF-8+BOM when the file is saved. If you don't want this change as part of this pull request, I can eliminate it. However, once made, there should not be further churn with respect to this.

File: src/VAP/VideoPipelineCore/VideoPipelineCore.csproj [hunk 3]
```diff
@@ -6,9 +6,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenCvSharp4" Version="4.0.0.20190108" />
-    <PackageReference Include="OpenCvSharp4.runtime.ubuntu.18.04-x64" Version="4.0.0.20190108" />
-    <PackageReference Include="OpenCvSharp4.runtime.win" Version="4.0.0.20190108" />
+    <PackageReference Include="OpenCvSharp4" Version="4.1.1.20191110" />
+    <PackageReference Include="OpenCvSharp4.runtime.ubuntu.18.04-x64" Version="4.1.1.20191023" />
+    <PackageReference Include="OpenCvSharp4.runtime.win" Version="4.1.1.20191110" />
```
Comment (sharwell) [inline src/VAP/VideoPipelineCore/VideoPipelineCore.csproj]: 📝 I'm not sure why there is a runtime mismatch between ubuntu and win, but it appears these are not always uploaded to NuGet as a pair.

File: src/VAP/BGSObjectDetector/BGSObjectDetector.csproj (hunk 1 above)
Comment (ycshu086) [reply src/VAP/BGSObjectDetector/BGSObjectDetector.csproj]: Good point. Can you please add the change to this PR?

**Review by ycshu086 — CHANGES_REQUESTED** [review]

File: src/VAP/BGSObjectDetector/BGSObjectDetector.csproj (hunk 1 above)
Comment (sharwell) [reply src/VAP/BGSObjectDetector/BGSObjectDetector.csproj]: Done!

**Pull request merged**